frontend/utility/OBSThemeVariable.hpp

```diff
@@ -28,6 +28,8 @@ struct OBSThemeVariable {
 		String, /* Raw string (e.g. color name, border style, etc.) */
 		Alias,  /* Points at another variable, value will be the key */
 		Calc,   /* Simple calculation with two operands */
+		Min,    /* Get the smallest of two Size or Number */
+		Max,    /* Get the largest of two Size or Number */
```

Comment:
I think it's fine to say "Sizes" and "Numbers", unless this is a style preference.

Reply:
Seemed more appropriate since it's referring to the actual enum names